### DISCUSSION TITLE
Antd Select not working with setValue

### DISCUSSION THREAD

**Discussion by ajaybeniwal** [react-hook-form/react-hook-form#1778] [Q&A] [answered]
We are using Controller component to wrap Antd select component , but when we call set value on form passing the name of control it does not set the display value of the select 

https://codesandbox.io/embed/hopeful-ramanujan-8v7jo?fontsize=14&hidenavigation=1&theme=dark

**Comment by bluebill1049** [comment]
1. it's super hard to read without format
2. please supply a codesandbox for your question

**Comment by ajaybeniwal** [comment]
And one more thing the nested component does not work either do we need to pass anything down to the component

**Comment by bluebill1049** [comment, marked as answer]
I am not a pro at AntD, but controller definitely works: https://codesandbox.io/s/modern-leaf-wikw5?file=/src/App.js

**Reply by bluebill1049** [reply]
wait i miss the setValue part, adding it now

**Reply by bluebill1049** [reply]
done and updated.
